dotCMS/src/main/java/com/dotmarketing/portlets/cmsmaintenance/ajax/CMSMaintenanceAjax.java

```diff
@@ -546,7 +553,13 @@ public void createXMLFiles() throws ServletException, IOException, DotDataExcept
 
 						if (list != null && list.size() > 0 && list
 								.get(0) instanceof Comparable) {
-							java.util.Collections.sort(list);
+						    try {
+						        java.util.Collections.sort(list);
+						    }
+						    catch(Exception e) {
+						        System.err.println(list);

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)